slackify-html.js

```diff
@@ -181,6 +194,9 @@ function walk(dom, nesting) {
                 if (suffixSpace) {
                   innerOutput += ' ';
                 }
+                if (el.name === 'h1' || el.name === 'h2' || el.name === 'h3') {
```

Comment:
if you move this logic up to line 174 where it checks the string for existing '*', you could just handle the header case there and used `replace` to string all '*' from the string first, then add the surrounding '*'.

Reply:
also, does this need to handle the 'h4' case as well?

Reply:
@jahlbornG good point! also, not sure about h4 but I'll just include it just in case